addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -949,7 +953,7 @@ registry.ScrollButton = registry.anchorSlide.extend({
         ev.preventDefault();
         const $nextSection = this.$el.closest('section').next('section');
         if ($nextSection.length) {
-            this._scrollTo($nextSection);
+            this._scrollTo($nextSection, 'true');
```

Comment:
Could you change the method so that the second parameter is optional and 'true' by default so that there is no need to change here?